foursight_core/app_utils.py

```diff
@@ -927,13 +925,20 @@ def queue_scheduled_checks(self, sched_environ, schedule_name, conditions=None):
             for environ in sched_environs:
                 # add the run info from 'all' as well as this specific environ
                 check_vals = copy.copy(check_schedule.get('all', []))
-                check_vals.extend(check_schedule.get(environ, []))
+                check_vals.extend(self.get_env_schedule(check_schedule, environ))
                 self.sqs.send_sqs_messages(queue, environ, check_vals)
         runner_input = {'sqs_url': queue.url}
         for n in range(4):  # number of parallel runners to kick off
             self.sqs.invoke_check_runner(runner_input)
         return runner_input  # for testing purposes
 
+    @classmethod
+    def get_env_schedule(cls, check_schedule, environ):
```

Comment:
I would docstring this saying something to the effect of: "in the new environment configuration, there are multiple aliases that refer to the same environment. This function ensures that when writing a check schedule you can refer to any of the aliases."

On a related note (not something for this PR), one thought that I'm just having now though is whether you could create an ambiguity by designating `ENV_NAME=cgap-devtest` and another `ENV_NAME=devtest`. Should we have some sort of protection for that thing? It might be useful to provide a "validation" command for the ecosystem setup that checks for such things.